root_locals.tf

```diff
@@ -86,4 +86,5 @@ locals {
   //Feature access blocks
   block_feature_closure_metadata     = true
   block_feature_descriptive_metadata = true
+  block_feature_view_history         = true
```

Comment:
I think it is safe to release the feature on intg and staging as it won't break the e2e tests, so you can set this as follows: `block_feature_view_transfers = local.environment == "prod" ? true : false`